frontend/src/container/WorkspaceContent.jsx

```diff
@@ -100,18 +100,23 @@ class WorkspaceContent extends React.Component {
   }
 
   async componentDidMount () {
-    const { workspaceList, match } = this.props
+    const { props } = this
 
     console.log('%c<WorkspaceContent> componentDidMount', 'color: #c17838')
 
     let wsToLoad = null
 
-    if (match.params.idws === undefined) {
-      if (workspaceList.length > 0) wsToLoad = workspaceList[0].id
+    if (props.match.params.idws === undefined) {
+      if (props.workspaceList.length > 0) wsToLoad = props.workspaceList[0].id
       else return
-    } else wsToLoad = match.params.idws
+    } else wsToLoad = props.match.params.idws
 
-    await this.loadContentList(wsToLoad)
+    try {
+      await this.loadContentList(wsToLoad)
+    } catch (error) {
+      console.log('error while loading content list in workspace content mount')
```

Comment:
Possibility to have error message in printed log ?

Reply:
You're right, done.